SimPEG/regularization/pgi.py

```diff
@@ -1195,7 +1195,7 @@ def __init__(
                     alpha_zz=alpha_zz,
                     mesh=self.regularization_mesh,
                     mapping=model_map * wire[1],
-                    weights=weights,
+                    weights={"user_weights": weights},
```

Comment:
Let's restore this to the original state. I can deal with PGI afterwards.
```suggestion
                    weights=weights,
```